met-web/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# pull official base image
+FROM node:13.12.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY package.json ./
+COPY package-lock.json ./
```

Comment:
is the package-lock needed ?

Reply:
its optional but it makes sure that the packages are being downloaded from the same source as when it was added